usaspending_api/awards/v2/data_layer/sql.py

```diff
@@ -1,67 +1,25 @@
-            faba.reporting_fiscal_period,
-            sum(faba.gross_outlay_amount_by_award_cpe) as current_fy_outlay
-        from eligible_file_c_records faba
-        where
-            faba.reporting_fiscal_period != 12 -- don't duplicate the year-end period's value if in unclosed period 01 or 02 (since there is no P01 submission)
-            and (faba.reporting_fiscal_year, faba.reporting_fiscal_period) in (
-            -- Most recent closed period
-                select c.submission_fiscal_year, c.submission_fiscal_month
-                from closed_periods c
-                order by c.submission_fiscal_year desc, c.submission_fiscal_month desc
-                limit 1
-            )
-            and (faba.gross_outlay_amount_by_award_cpe is not null and faba.gross_outlay_amount_by_award_cpe != 0) -- negative or positive balance
-        group by
-            faba.disaster_emergency_fund_code,
-            faba.reporting_fiscal_year,
-            faba.reporting_fiscal_period
-    )
-    select
-        faba.disaster_emergency_fund_code,
-        coalesce(sum(faba.transaction_obligated_amount), 0) as obligated_amount,
-        coalesce(sum(ffy.prior_fys_outlay), 0) + coalesce(sum(cfy.current_fy_outlay), 0) as total_outlay
-    from eligible_file_c_records faba
-    left join fy_final_outlay_balances ffy on ffy.disaster_emergency_fund_code = faba.disaster_emergency_fund_code and faba.rownum = 1
-    left join current_fy_outlay_balance cfy on cfy.disaster_emergency_fund_code = faba.disaster_emergency_fund_code and faba.rownum = 1
-    group by faba.disaster_emergency_fund_code
-    order by obligated_amount desc
+    SELECT
+        DISTINCT disaster_emergency_fund_code,
+        COALESCE(sum(CASE WHEN latest_closed_period_per_fy.is_quarter IS NOT NULL THEN faba.gross_outlay_amount_by_award_cpe END), 0) AS total_outlay,
+        COALESCE(sum(faba.transaction_obligated_amount), 0) AS obligated_amount
+    FROM
+        financial_accounts_by_awards faba
+    INNER JOIN disaster_emergency_fund_code defc
+        ON defc.code = faba.disaster_emergency_fund_code
+        AND defc.group_name = 'covid_19'
+    INNER JOIN submission_attributes sa
+        ON faba.submission_id = sa.submission_id
+        AND sa.reporting_period_start >= '2020-04-01'
+    LEFT JOIN (
+        SELECT   submission_fiscal_year, is_quarter, max(submission_fiscal_month) AS submission_fiscal_month
+        FROM     dabs_submission_window_schedule
+        WHERE    submission_reveal_date < now() AND period_start_date >= '2020-04-01'
+        GROUP BY submission_fiscal_year, is_quarter
+    ) AS latest_closed_period_per_fy
+        ON latest_closed_period_per_fy.submission_fiscal_year = sa.reporting_fiscal_year
+        AND latest_closed_period_per_fy.submission_fiscal_month = sa.reporting_fiscal_period
+        AND latest_closed_period_per_fy.is_quarter = sa.quarter_format_flag
+    WHERE {award_id_sql}
+    GROUP BY disaster_emergency_fund_code
```

Comment:
Was the sort lost, or is it unnecessary?

Reply:
I believe it's unnecessary, I don't recall it being used for anything, but I can put it back if we think it's important

Reply:
If I'm reading the code correctly, the order of the results sets the order of the outlay and obligation arrays in the API response. That order _shouldn't_ matter but I don't know how they are used. If there isn't a noticeable performance impact I recommend keeping it, definitely wouldn't sacrifice performance for it.

Reply:
Performance seems unchanged, so I've added it back in just to be consistent

Reply:
Ok, thanks!